### ISSUE TITLE
UX issues [Collectives 2.15]: Unreadable from mobile

### ISSUE THREAD

**Issue by estux** [nextcloud/collectives#1577]
**Describe the bug**
Opening the Collective from public shared link on mobile browser the page is not centered, as the page scrolls horizontally a lot. If you try to move the text to the center it automatically opens the left sidebar so the experience is terrible to the point of making near to impossible to read the content.

**To Reproduce**
1. Open a public shared Collective on mobile
2. Close the section Recent Pages (another UX issue is that Recent Page cover totally the landing page making it near to impossible to read the landing page content and it's not intuitive to know you should close the Recent page section)
3. See the text toward the right part of the screen, half-visible
4. Try to move the text to the center (on touch screen)
5. It automatically opens the left sidebar covering the content
6. If you close the sidebar the text comes back to an uncomfortable position again

**Expected behavior**
I should be able to read the text properly and having it properly placed. Access to content should be straightforward.

**Screenshots**

**Server details:**
- Collectives app version: 2.15 (just updated)
- Nextcloud version: 29.0.8
- PHP Version: 8.2.14
- Database: mysql 10.5.26

**Client details:**
 - OS: Android 13 (Samsung UI 5)
 - Browser: Fennec (Firefox mobile)
 - Browser version: 132.0
 - Device: Samsung S20FE

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

</details>

**Comment by estux** [comment]
I will open another issue for the Recent Pages problem on mobile (as I need to tell before to all the people to close it to see the content).

**Comment by lattaai3** [comment]
Hello, I tried to fix the issue

This is what I did:

This change addresses the issue of content not being centered and difficult to read on mobile devices when accessing a public shared Collective. It modifies the CSS to ensure proper centering and improves the mobile layout.

I also created pull request: https://github.com/nextcloud/collectives/pull/1579


> [!CAUTION]
> Disclaimer: This fix was created by Latta AI and you should never merge before you check the correctness of generated code!
>
> The fix provided by Latta AI might not be complete and it can serve as an inspiration.

---
This bug was fixed for free by Latta AI - https://latta.ai/ourmission

If you no longer want Latta AI to attempt fixing issues on your repository, you can block this account.
